### PR TITLE
[update] screenshots rebrand second set

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

made the URL shorter: https://assets.timescale.com/docs/images
Updated the images and changed the format

# Links

Fixes https://github.com/timescale/docs-private/issues/126

https://docs.google.com/spreadsheets/d/1nTPlL2gXApmfNEi0jyPnuGRTfK-vq14bgBDyVy9ZUVE/edit#gid=0

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: _partials/_cloud-create-service.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Each service contains a single database, named `tsdb`.
       </p>
       <img
         class="main-content__illustration"
-        src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-create-service-demo.png"
+        src="https://assets.timescale.com/docs/images/tsc-create-service-demo.png"
```
Comment (charislam) [inline _partials/_cloud-create-service.md:42]: Looks like this one wasn't updated?

File: use-timescale/compression/about-compression.md [hunk 2]
```diff
@@ -162,11 +162,7 @@ candidates, and are eventually compressed when they become old enough according
 to the compression policy.
 
 <img
-<<<<<<< HEAD:use-timescale/compression/about-compression.md
-class="main-content__illustration" src="<https://assets.timescale.com/images/diagrams/compression_diagram.png"alt="Compression>>
-=======
-class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/compression_diagram.webp"alt="Compression>
->>>>>>> 375040ef14bda3a2bd80aa64e1a3a498905b1c66:self-hosted/compression/index.md
+class="main-content__illustration" src="https://assets.timescale.com/docs/images/compression_diagram.webp"alt="Compression>
```
Comment (charislam) [inline use-timescale/compression/about-compression.md:165]: missing closing dbl quote

**Review by charislam — COMMENTED** [review]
Hi Rajie, looks like most of these are missing width and height. I'm fine with it if you want to do them in a second pass in order to get this PR out the door

**Comment by Rajakavitha1** [comment]
> Hi Rajie, looks like most of these are missing width and height. I'm fine with it if you want to do them in a second pass in order to get this PR out the door

Yes @charislam !!! I was planning on doing the width and height all at once.

File: use-timescale/compression/about-compression.md (hunk 2 above)
Comment (Rajakavitha1) [reply use-timescale/compression/about-compression.md]: Thank you for spotting this. It has now been fixed.

**Comment by Rajakavitha1** [comment]
Hi @charislam !!! Just incorporated the changes that you suggested. Please approve so that we can get this out of the door.

**Comment by Rajakavitha1** [comment]
> All seems good, but we need to put the image size in as well.

Hi @Loquacity  can we have the image size all at once when we have all the new rebranded images. That will ensure that we don't miss any images.

**Comment by Loquacity** [comment]
> > All seems good, but we need to put the image size in as well.
> 
> Hi @Loquacity can we have the image size all at once when we have all the new rebranded images. That will ensure that we don't miss any images.

Sure, that works 👍

**Review by Loquacity — APPROVED** [review]

**Pull request merged**